raspberry/bzzz/sensors/evo_time_of_flight.py

```diff
@@ -108,6 +108,15 @@ def __exit__(self, *args):
         time.sleep(0.05)
         if self.__log_file is not None:
             self.__logger.save_to_csv(self.__log_file)
+    
+    def __updateToF__(self):
```

Comment:
Do we need this function? 
is this not more or less the same as `self.__values_cache` only limiting the length of the cache

Reply:
why would we want to limit the length of the cache?

Reply:
@pdavid747 To prevent the cache from taking up too much memory.

Reply:
@jamie-54 If we don't need to worry about memory usage issue I can remove this function.

Reply:
I don't think we need this function since we have the data logger